extensions/widgets/segmented/segmented.lcb

```diff
@@ -425,11 +424,11 @@ public handler OnLoad(in pProperties as Array)
 	put pProperties["showFrameBorder"]	into mShowFrameBorder
 	
 	if "selectedColor" is among the keys of pProperties then
-		put stringToColor(pProperties["selectedColor"]) into mSelectedColor
+		set property "hiliteColor" of my script object to pProperties["selectedColor"]
 	end if
 	
 	if "color" is among the keys of pProperties then
-		put stringToColor(pProperties["color"]) into mColor
+		set property "backColor" of my script object to pProperties["selectedColor"]
```

Comment:
Just noticed this - these keys are wrong.

This will probably change when `my ... paint` is implemented anyway.